packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.js

```diff
@@ -42,6 +49,14 @@ export default class QuanticCitation extends NavigationMixin(LightningElement) {
   isInitialRender = true;
   /** @type {string} */
   salesforceRecordUrl;
+  /** @type {boolean} */
+  isWithTextFragment = false;
+
+  connectedCallback() {
+    const fileType = this.citation?.fields?.filetype;
+    this.isWithTextFragment =
+      !this.disableCitationAnchoring || fileType !== 'html' || !this.text;
```

Comment:
This condition is wrong 🤔
it should be true when `!this.disableCitationAnchoring && fileType === 'html' && !!this.text`

Seeing that the unit tests did not catch that means that the test coverage needs to be improved. we should test the different combination of the terms composing this condition as its logic is essential the inner working of the component.